main_controller.py

```diff
@@ -0,0 +1,88 @@
+"""
+Description: 
+
+This script is the main controller for the Dephy EB-51 Exoskeletons.
+It is responsible for initializing the exoskeletons and running the main control loop.
+
+NOTE: 
+This script deliberately uses the downgraded 8.0.1 flexsea library.
+The newer flexsea libraries have an integer overflow issue, preventing 
+new sensor data from being received. This issue only exists for the exoskeleton devices
+(not the actpacks). The 8.0.1 library is stable with a Raspberry Pi 4B with a 32-bit OS, 
+using python 3.9.
+
+Date: 04/29/2025
+Author(s): Nundini Rawal
+"""
+
+from opensourceleg.logging import Logger, LogLevel
+from opensourceleg.utilities import SoftRealtimeLoop
+
+from exoboots import DephyExoboots
+from src.utils.actuator_utils import create_actuators
+from src.utils.filing_utils import get_logging_info
+from src.settings.constants import(
+    BAUD_RATE,
+    FLEXSEA_FREQ,
+    LOG_LEVEL
+)
+
+def track_variables_for_logging(logger: Logger) -> None:
```

Comment:
This function should look for the number of active sides and then track those variables (and not just be used for tracking the left exoboot values). Depending upon if only the left, right or both are active it should track those variables for logging.